Gemfile

```diff
@@ -54,7 +62,10 @@ group :development do
 
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
+  gem 'pry', '~> 0.10.4'
+  gem 'better_errors'
 
+  gem 'rb-readline', platforms: :ruby, install_if: os_is(/darwin/)
```

Comment:
Why not something more simplier like:
```ruby
  gem 'rb-readline', platforms: :ruby, require: RUBY_PLATFORM =~ /darwin/i ? 'rb-readline' : false
```
(is platforms needed even?)

(RUBY_PLATFORM contains your OS version for example mine is RUBY_PLATFORM ==> "x86_64-linux")
Then you don't need the helper os method above?